selfdrive/car/nissan/values.py

```diff
@@ -7,6 +7,7 @@
 class CAR:
   XTRAIL = "NISSAN X-TRAIL 2017"
   LEAF = "NISSAN LEAF 2018"
+  LEAF_IC = "NISSAN LEAF 2018"
```

Comment:
this needs to be unique

Reply:
IC is short for "instrument cluster" as this version of the leaf does not have the adapter behind the glovebox but behind the instrument cluster. I am not yet sure if a differentiation between the glovebox and the IC version is even needed. If I comment one line at least the IC version works as expected.

Reply:
@adeebshihadeh done, also added a route which can not yet be found by ci